src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.hpp

```diff
@@ -175,6 +175,44 @@ class SinglePhaseBase : public FlowSolverBase
                      CRSMatrixView< real64, globalIndex const > const & localMatrix,
                      arrayView1d< real64 > const & localRhs ) = 0;
 
+  /**
+   * @brief assembles the flux terms for all cells for the poroelastic case
+   * @param time_n previous time value
+   * @param dt time step
+   * @param domain the physical domain object
+   * @param dofManager degree-of-freedom manager associated with the linear system
+   * @param localMatrix the system matrix
+   * @param localRhs the system right-hand side vector
+   * @param jumpDofKey dofKey of the displacement jump
+   */
+  virtual void
+  assemblePoroelasticFluxTerms( real64 const time_n,
```

Comment:
These things look a bit hacky, but I don't have a concrete refactoring suggestion at the moment. Unless somebody has one, let's just merge as is and refactor later.